.github/workflows/testing.yml

```diff
@@ -72,7 +72,10 @@ jobs:
       # Only run vet for 'vet' runs.
       - name: Run vet.sh
         if: ${{ matrix.type == 'vet' }}
-        run: ./vet.sh -install && ./vet.sh
+        run: |
+          ./vet.sh -install && ./vet.sh
+          cd ${GITHUB_WORKSPACE}/security/advancedtls && go vet -all ./...
+          cd ${GITHUB_WORKSPACE}/security/authorization && go vet -all ./...
```

Comment:
These checks should go into `vet.sh` instead.  Devs *should* be running `vet.sh` locally to vet their changes, not github actions.

Reply:
When implemented, how about something like:

```sh
for x in $(find . -name 'go.mod'); do
  pushd $(dirname $x)
  go mod tidy # replacing the existing go mod tidy per-go.mod we already do
  go vet -all ./...
  # anything else per-module? are gofmt, goimports, golint, staticcheck, etc also module-bound?
  popd
done
```

Reply:
Thanks for the code snippet! It appears to be very useful :)